Sofa/Component/Constraint/Projective/src/sofa/component/constraint/projective/FixedTranslationConstraint.inl

```diff
@@ -113,32 +113,32 @@ static inline void clearPos(type::Vec<6,T>& v)
 }
 
 template <class DataTypes> template <class DataDeriv>
-void FixedTranslationConstraint<DataTypes>::projectResponseT(const core::MechanicalParams* /*mparams*/, DataDeriv& res)
+void FixedTranslationConstraint<DataTypes>::projectResponseT(DataDeriv& dx,
+    std::function<void(DataDeriv&, const unsigned int)> clear)
 {
-    const SetIndexArray & indices = f_indices.getValue();
-
-    if (f_fixAll.getValue() == true)
+    if (f_fixAll.getValue())
     {
-        for (Size i = 0; i < l_topology.get()->getNbPoints(); ++i)
+        for (unsigned int i = 0; i < dx.size(); i++)
```

Comment:
I doubt that the type of `dx.size()` is `unsigned int`. Could you check?

Reply:
I doubt too...